### PR TITLE
Replace remapped concepts

### PR DESCRIPTION
**Pull request by chrzyki**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

This PR replaces most concepts (apart from a few special lists/cases which will be done separately) with their replacement counterparts. See also #810, #880, and #816. Given the number of changes, I'd appreciate a moderated workflow on this one. @Schweikhard, would you mind overlooking this? @LinguList, I'd also be grateful if you could have a look at this if you have time.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
@Schweikhard is on vacation for two weeks. But I'm happy to step in and moderate this PR.

**Comment by chrzyki** [comment]
Ah, thanks! Yes, that would be very kind, thanks @AnnikaTjuka.

**Comment by AnnikaTjuka** [comment]
@Kristina-Pianykh would you mind reviewing this PR?

**Comment by LinguList** [comment]
Please wait until I approve. Remind me tomorrow if you do not hear from me on this.

**Comment by AnnikaTjuka** [comment]
Sure, will do.

File: concepticondata/conceptlists/Steinthal-1875-1549.tsv [hunk 1]
```diff
@@ -1456,7 +1456,7 @@ Steinthal-1875-1549-1454	1454	Die Possessiva	dein, deins	your	732	YOUR (SINGULAR
 Steinthal-1875-1549-1455	1455	Die Possessiva	sein, seins	his	2473	HIS (GENITIVE OF HE)
 Steinthal-1875-1549-1456	1456	Die Possessiva	ihr, ihrs	her	2474	HER (GENITIVE OF SHE)
 Steinthal-1875-1549-1457	1457	Die Possessiva	unser, unsers	our	780	OUR
-Steinthal-1875-1549-1458	1458	Die Possessiva	euer, euers	your	2091	YOUR
+Steinthal-1875-1549-1458	1458	Die Possessiva	euer, euers	your	732	YOUR (SINGULAR)
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists/Steinthal-1875-1549.tsv:1459]: Why not YOUR (PLURAL) 2274?

Comment (chrzyki) [reply concepticondata/conceptlists/Steinthal-1875-1549.tsv:1459]: My mistake, thanks for spotting.

**Comment by AnnikaTjuka** [comment]
Thanks, @Kristina-Pianykh and @LinguList for reviewing this PR. Feel free to merge, @chrzyki .

**Pull request merged**